### DISCUSSION TITLE
Fixed `MqttRetry` timeout, no or less automatic increase

### DISCUSSION THREAD

**Discussion by schildbach** [arendst/Tasmota#20471] [Q&A] [answered]
Currently if I set a `MqttRetry` value or keep the default of 10 seconds, it keeps increasing the timeout with each try. I'd like to prevent that, or at least limit the maximum retry time to a much lower value than 120 seconds. How can I do that?

**Comment by sfromis** [comment, marked as answer]
That logic to increase the interval is very deliberate, to avoid senseless hammering during extended outages. And I see no existing option for changing that scheme.

**Reply by schildbach** [reply]
I agree it's nice to have. But for my usecase it's obstructive. And 10s is not exactly "senseless hammering" if you've got only a handful of MQTT clients.

**Reply by sfromis** [reply]
Every 10th second (or faster as you'd like to have) for hours is what I'd call "senseless hammering". Sure, it should not bog the whole system down, but still.

What I consider a basic part of a setup is to have the MQTT broker running at least 99.99%

**Reply by barbudor** [reply]
You can use a rule to trigger a restart if you don't get mqtt within 30 sec. That would shorten the delay for reconnection attempt at the cost of much more 'network traffic
